src/main/resources/application.properties

```diff
@@ -17,7 +17,15 @@ alert.accumulator.cron=0 0/1 * 1/1 * *
 alert.digest.daily.cron=0 0 0 1/1 * ?
 alert.digest.realtime.cron=0 0/1 * 1/1 * *
 
-spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.url=jdbc:h2:file:~/test
```

Comment:
Let's change this to alertdb and see if we can change the path to the directory running the application. 

Also how did you test this?  The DataSourceConfig was needed because Spring batch had some conflicts with the JPA repository, specifically the transaction manager it was using.

Reply:
I removed the datasource file as it mostly was configuring springs default in memory testdb. The thing that was creating this was the DataSource bean specifically. I can add the TransactionManager back but I would like to test your exact scenario first to see if we still have this conflict, or if just the in memory DB was the root cause of the issue.